test/httpRequestAction.spec.js

```diff
@@ -38,6 +38,35 @@ describe('httpRequest action', () => {
     nock.cleanAll();
   });
 
+  describe('API key credentials', () => {
+    const msg = {
+      data: {
+        url: 'http://example.com',
+      },
+    };
+    const cfg = {
+      reader: {
+        url: '$$.data.url',
+        method: 'POST',
+        headers: [],
+      },
+      auth: {},
+    };
+
+    it('should accept an authorization header with a bearer token', async () => {
+      cfg.headerName = 'authorization';
```

Comment:
Can you capitalize Authorization to make it like your existing scenario?

Reply:
It looks like you changed the wrong line. I expect Authorization at line 57 to have the first letter capitalized because that's how it would be retrieved from the secret.

Reply:
We can’t check for Authorization w/ a capital A here because the headers are automatically formatted to be lowercase. Authorization here is undefined.

Reply:
Oh, I misunderstood what you meant. Ignore that, I’ll fix it.